test/qml/tst_positioning.qml

```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.3
+import QtTest 1.0
+
+import org.qfield 1.0
+
+import QFieldControls 1.0
+
+TestCase {
+    name: "Positioning"
+
+    Positioning {
+        id: positioning
+        deviceId: 'udp:localhost:1958'
+        active: true
+        ellipsoidalElevation: true
+
+        coordinateTransformer: CoordinateTransformer {
+          id: coordinateTransformer
+          destinationCrs: CoordinateReferenceSystemUtils.wgs84Crs()
+          transformContext: CoordinateReferenceSystemUtils.emptyTransformContext()
+          deltaZ: 0
+          skipAltitudeTransformation: false
+          verticalGrid: ''
+        }
+    }
+
+    function test_ellipsoidalElevation() {
+        coordinateTransformer.deltaZ = 0
+        coordinateTransformer.verticalGrid = ''
+        // wait a few seconds so positioning can catch some NMEA strings
+        wait(5000)
+
+        // the elevation in the NMEA stream goes between 320 to 322, and the ellispodal adjustment is -26.0 meters
+        // we therefore simply check whether we are int the 200s value range, which indicates ellispodal elevation is
+        // being returned
+        compare(Math.floor(positioning.positionInformation.elevation / 100), 2)
+    }
+
+    function test_deltaZ() {
+        coordinateTransformer.deltaZ = -100
+        coordinateTransformer.verticalGrid = ''
+        // wait a few seconds so positioning can catch some NMEA strings
+        wait(5000)
```

Comment:
That's a lot of waits in all these tests... Can we reduce this to <1s, e.g. by increasing the server rate?

Reply:
@m-kuhn , I shall test and adjust :)

Reply:
@m-kuhn , double server rate and decreased wait by half for each test. Tests were failing (either osx or both osx and linux at times) using lower values.